libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -1185,4 +1313,42 @@ public void onReceive(Context context, Intent intent) {
         }
     }
 
+    /**
+     * Action handler in dev support dialog
+     */
+    public interface DevActionHandler {
+        /**
+         * Triggered in case when user select the action
+         */
+        void onSelected();
+    }
+
+    /**
+     * Get BuildConfig.DEBUG by reflection (since it's only available in the app project)
+     * @return true if app's BuildConfig.DEBUG is true
+     */
+    private boolean isDebugBuild() {
+        return ((Boolean) getBuildConfigValue(getAppContext(), "DEBUG")).booleanValue();
+    }
+
+    /**
+     * Gets a field from the project's BuildConfig.
+     * @param context       Used to find the correct file
+     * @param fieldName     The name of the field-to-access
+     * @return              The value of the field, or {@code null} if the field is not found.
+     */
+    private Object getBuildConfigValue(Context context, String fieldName) {
```

Comment:
We have to use reflection because BuildConfig is only available in the app project